pkg/apps/application-list.jsx

```diff
@@ -159,6 +172,9 @@ export const ApplicationList = ({ metainfo_db, appProgress, appProgressTitle, ac
             {comps.length == 0
                 ? <EmptyStatePanel title={ _("No applications installed or available.") } />
                 : <PageSection>
+                    {dataPackagesInstalled == false && <Alert variant="warning"
```

Comment:
Just one last comment, but having this inside `comps.length` sounds wrong, since we'll never let the user know that appstream must be installed if they have none of the optional applications present.